### PR TITLE
feat: add shortcut for writing stdout and stderr in the cli package

### PR DESCRIPTION
**Pull request by yolocs**

### CODE REVIEW

File: cli/command.go [hunk 1]
```diff
@@ -61,10 +61,14 @@ type Command interface {
 	// Stdout returns the stdout stream. SetStdout sets the stdout stream.
 	Stdout() io.Writer
 	SetStdout(w io.Writer)
+	// Outf is a shortcut to write to [Command.Stdout].
```
Comment (sethvargo) [inline cli/command.go:64]:
```suggestion

	// Outf is a shortcut to write to [Command.Stdout].
```

File: cli/command.go [hunk 2]
```diff
@@ -61,10 +61,14 @@ type Command interface {
...
+	Outf(format string, a ...any)
 
 	// Stderr returns the stderr stream. SetStderr sets the stderr stream.
 	Stderr() io.Writer
 	SetStderr(w io.Writer)
+	// Errf is a shortcut to write to [Command.Stderr].
```
Comment (sethvargo) [inline cli/command.go:70]:
```suggestion

	// Errf is a shortcut to write to [Command.Stderr].
```

File: cli/command_test.go [hunk 3]
```diff
@@ -314,7 +314,7 @@ func (c *TestCommand) Run(ctx context.Context, args []string) error {
 	}
 
 	if v := c.Output; v != "" {
-		fmt.Fprint(c.Stdout(), v)
+		c.Outf(v)
```
Comment (sethvargo) [inline cli/command_test.go:317]: I think there might be a few examples to clean up too?

**Review by sethvargo — APPROVED** [review]

**Comment by sethvargo** [comment]
Thanks for doing this!

File: cli/command_test.go (hunk 3 above)
Comment (yolocs) [reply cli/command_test.go:317]: Done

**Pull request merged**